### PR TITLE
Add Dendroscope 3.6.3

### PR DESCRIPTION
**Pull request by romainstuder**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].
- [ ] Attempted to find an [appcast].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256
[appcast]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md

### CODE REVIEW

**Comment by romainstuder** [comment]
The line `installer script: 'Dendroscope Installer.app/Contents/MacOS/JavaApplicationStub'` opens a GUI interface which guide user in the installation process. While it works on my laptop, it makes the travis-ci to stall, with the alert: `WARNING: GL pipe is running in software mode`. Any suggestions? Otherwise I could put `installer manual: 'Dendroscope Installer.app/Contents/MacOS/JavaApplicationStub'` and the user has to manually open the link. But I found this less clean.

File: Casks/dendroscope.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  version '3.6.3'
+  sha256 '2b9557232149da9d6d2fa0d07bd98b21836141e2759ca92986e5d7a3e21d0bde'
+
+  url "https://ab.inf.uni-tuebingen.de/data/software/dendroscope3/download/Dendroscope_macos_#{version.dots_to_underscores}.dmg"
+  name 'Dendroscope'
+  homepage 'https://ab.inf.uni-tuebingen.de/software/dendroscope/welcome.html'
+
+  installer script: 'Dendroscope Installer.app/Contents/MacOS/JavaApplicationStub'
+
+  uninstall script: '/Applications/Dendroscope/Dendroscope Uninstaller.app/Contents/MacOS/JavaApplicationStub'
```
Comment (vitorgalvao) [inline Casks/dendroscope.rb]: Does this one call a GUI as well?

File: Casks/dendroscope.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+cask 'dendroscope' do
+  version '3.6.3'
+  sha256 '2b9557232149da9d6d2fa0d07bd98b21836141e2759ca92986e5d7a3e21d0bde'
+
+  url "https://ab.inf.uni-tuebingen.de/data/software/dendroscope3/download/Dendroscope_macos_#{version.dots_to_underscores}.dmg"
+  name 'Dendroscope'
+  homepage 'https://ab.inf.uni-tuebingen.de/software/dendroscope/welcome.html'
+
+  installer script: 'Dendroscope Installer.app/Contents/MacOS/JavaApplicationStub'
```
Comment (vitorgalvao) [inline Casks/dendroscope.rb]: We have other casks with `JavaApplicationStub` that use arguments to make a GUI-less install. Reference them to see if the same works here.

File: Casks/dendroscope.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
+cask 'dendroscope' do
+  version '3.6.3'
+  sha256 '2b9557232149da9d6d2fa0d07bd98b21836141e2759ca92986e5d7a3e21d0bde'
+
+  url "https://ab.inf.uni-tuebingen.de/data/software/dendroscope3/download/Dendroscope_macos_#{version.dots_to_underscores}.dmg"
+  name 'Dendroscope'
+  homepage 'https://ab.inf.uni-tuebingen.de/software/dendroscope/welcome.html'
```
Comment (vitorgalvao) [inline Casks/dendroscope.rb]:
```suggestion
  homepage 'https://ab.inf.uni-tuebingen.de/software/dendroscope/'
```

Comment (romainstuder) [reply Casks/dendroscope.rb]: Thanks!

File: Casks/dendroscope.rb (hunk 2 above)
Comment (romainstuder) [reply Casks/dendroscope.rb]: Thanks for the suggestion! I followed your advice and managed to build a working formula.

File: Casks/dendroscope.rb [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+  preflight do
```
Comment (vitorgalvao) [inline Casks/dendroscope.rb]: Don’t use `preflight`, but `installer script:`, [like so](https://github.com/Homebrew/homebrew-cask/blob/4184d50d2d7fa4afb62e6c7eb6761ae5ac417c88/Casks/thinkorswim.rb).

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Comment by vitorgalvao** [comment]
Incorrect indentations. Run the `fix` line again.

File: Casks/dendroscope.rb (hunk 4 above)
Comment (romainstuder) [reply Casks/dendroscope.rb]: Thansk, this is now fixed.

**Comment by vitorgalvao** [comment]
File is formatted with tabs instead of spaces; indentation in the `installer` and `uninstall` scripts is wrong. `brew cask style --fix` should correct all of those.

**Comment by romainstuder** [comment]
> File is formatted with tabs instead of spaces; indentation in the `installer` and `uninstall` scripts is wrong. `brew cask style --fix` should correct all of those.

For some reasons, `brew cask style --fix` is not working on my Mac:

In Gemfile:
  mechanize was resolved to 2.7.6, which depends on
    http-cookie was resolved to 1.0.3, which depends on
      domain_name was resolved to 0.5.20190701, which depends on
        unf was resolved to 0.1.4, which depends on
          unf_ext


In Gemfile:
  ronn was resolved to 0.7.3, which depends on
    hpricot

In Gemfile:
  rubocop-performance was resolved to 1.4.1, which depends on
    rubocop was resolved to 0.74.0, which depends on
      jaro_winkler

**Pull request merged**